db/mysql/client.go

```diff
@@ -0,0 +1,122 @@
+package mysql
+
+import (
+	"database/sql"
+	//	"database/sql"
+	"fmt"
+	guuid "github.com/satori/go.uuid"
+	"strings"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/jmoiron/sqlx"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+
+	"github/wasimkhan042/uStore-Auth/config"
+	"github/wasimkhan042/uStore-Auth/db"
+	"github/wasimkhan042/uStore-Auth/models"
```

Comment:
un organised imports

Reply:
Opps, I really forgot about your last comment about imports.... will try best to not repeat this again... Thanks for your thoroughly review.